lte/protos/pipelined.proto

```diff
@@ -200,36 +200,19 @@ message PacketDropTableId {
 // source file: https://www.etsi.org/deliver/etsi_ts/129200_129299/129244/15.08.00_60/ts_129244v150800p.pdf
 //
 
-message NodeID {
-  enum NodeIDType {
-    IPv4 = 0;
-    IPv6 = 1;
-    FQDN = 2;
-    }
-  NodeIDType node_id_type = 1;
-  string node_id = 2;
-}
 
 //ToDo latest state change will reflect in upcoming proto
 message Fsm_state {
   enum FsmState {
-        SESSION_ACTIVE                        = 0;
-        SESSION_TERMINATED                    = 4;
-        SESSION_RELEASED                      = 6;
-        CREATING                              = 7;
-        CREATED                               = 8;
-        ACTIVE                                = 9;
-        INACTIVE                              = 10;
-        RELEASE                               = 11;
+    UNUSED       = 0;
+    //Adhering to current refactored converged sessiond states
+    CREATING     = 7;
+    CREATED      = 8;
+    ACTIVE       = 9;
+    INACTIVE     = 10;
+    RELEASE      = 11;
```

Comment:
is it possible to reset the identifiers and start from 1? 
Per my understanding FsmState is not used by any of our systems on production yet, so no backwards compatibility

Reply:
@gurrapualt Can you please elaborate reason for using these enums with the above indexing ?